setup.cfg

```diff
@@ -60,3 +60,4 @@ ara.cli =
     host show = ara.cli.host:HostShow
     stats list = ara.cli.stats:StatsList
     stats show = ara.cli.stats:StatsShow
+    static generate = ara.cli.static:StaticGenerate
```

Comment:
Why do we have a two-level command here?  How about just `ara generate` or `ara report`...